packages/testing/src/cypress/setup.ts

```diff
@@ -23,6 +24,8 @@ type ClerkSetupParams = {
  * @throws An error if the Cypress config object is not provided.
  */
 export const clerkSetup = async ({ config, options }: ClerkSetupParams) => {
+  experimentalConsoleWarning();
```

Comment:
Can we inline this instead of creating a separate function? If we don't use it in another place it's not really necessary

Reply:
We can, although I did it this way so it would be cleaner, but I can change it.